spec/requests/chatops/slack_slash_command_list_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "rails_helper"
+
+RSpec.describe "SlackSlashCommandListController", type: :request do
+  scenario "responds with default message when no channels configured" do
+    allow(Matchmaking).to receive(:config).and_return({})
+
+    slack_signing_secret = Slack::Events.config.signing_secret
+    timestamp = Time.zone.now.to_i
+    request_body = "token=#{slack_signing_secret}&team_id=T02PF6RHYSY&team_domain=testdouble-hq&channel_id=C02NYBB3VPH&channel_name=some-channel&user_id=U02PRHH0XEV&user_name=cliff.pruitt&command=%2Fdoubleup&text=list&api_app_id=A02PD0DUE03&is_enterprise_install=false&response_url=https%3A%2F%2Fhooks.slack.com%2Fcommands%2FT02PF6RHYSY%2F2823421496992%2F0WC0HfWeGJpHetxmF8yUmawo&trigger_id=2801968477828.2797229610916.883001cf5008d6d02fd58a3cf70f449e"
```

Comment:
This is gonna be tech debt. It's a pain to edit these when a param changes, but `Hash#to_param` does sorting that changes the request body so the HMAC signature isn't the same. I'd like to add a test helper in to manage this stuff but kinda just want to prioritize getting some features out.